### PR TITLE
Text css

### PR DESCRIPTION
**Pull request by levogirar**
- fix admin box-table bug
- minor changes

### CODE REVIEW

File: Templates/admin menu.php [hunk 1]
```diff
@@ -39,5 +39,5 @@
 			?>
 		</section>
 	</div>
-	Wuwana v2.1.5
+	<div id="version"> Wuwana v2.1.6 - text-css </div>
```
Comment (Nils85) [inline Templates/admin menu.php:42]: Why did you add "text-css" ??

File: static/dhtml/main.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ function showMenu()
 
 // To fix bug of filter menu not appearing after table view
 
-let menu = document.getElementById("menu");
+var menu = document.getElementById("menu");
```
Comment (Nils85) [inline static/dhtml/main.js:66]: 👍

File: Templates/admin menu.php (hunk 1 above)
Comment (levogirar) [reply Templates/admin menu.php:42]: to know which was the latest branch. Is it not a good idea?

Comment (Nils85) [reply Templates/admin menu.php:42]: No

**Review by Nils85 — APPROVED** [review]
